package.json

```diff
@@ -1,126 +1,58 @@
+    " ## ### ##    ##       ##    #########    ###             ##  ######  ",
+    " ##     ##    ##       ##    ##     ##   ## ##      ##    ##       ## ",
+    " ##     ##    ##       ##    ##     ##  ##   ##     ##    ## ##    ## ",
+    " ##     ##    ##       ##    ##     ## ##     ##     ######   ######  "
+  ],
   "keywords": [
     "MythX",
     "Smart Contract",
     "Ethereum",
     "Blockchain",
     "Security"
   ],
-  "main": "dist/mythxjs.umd.js",
-  "module": "dist/mythxjs.es5.js",
-  "typings": "dist/types/mythxjs.d.ts",
-  "files": [
-    "dist"
-  ],
-  "author": "Eswara Sai <eswarasai.v@gmail.com>",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/ConsenSys/mythxjs.git"
-  },
+  "author": "Mirko Garozzo",
+  "homepage": "https://github.com/ConsenSys/mythxjs",
+  "repository": "https://github.com/ConsenSys/mythxjs.git",
+  "bugs": "https://github.com/ConsenSys/mythxjs/issues",
   "license": "MIT",
-  "engines": {
-    "node": ">=8.0.0"
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "browser": {
+    "fs": false
   },
   "scripts": {
-    "lint": "tslint  --project tsconfig.json -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
-    "prebuild": "rimraf dist",
-    "build": "tsc --module commonjs && rollup -c rollup.config.ts && typedoc --out docs --target es6 --theme minimal --mode file src",
-    "start": "rollup -c rollup.config.ts -w",
-    "test": "jest --coverage",
-    "test:watch": "jest --coverage --watch",
-    "test:prod": "npm run lint && npm run test -- --no-cache",
-    "deploy-docs": "ts-node tools/gh-pages-publish",
-    "report-coverage": "cat ./coverage/lcov.info | coveralls",
-    "commit": "git-cz",
-    "semantic-release": "semantic-release",
-    "semantic-release-prepare": "ts-node tools/semantic-release-prepare",
-    "precommit": "lint-staged",
-    "travis-deploy-once": "travis-deploy-once"
-  },
-  "lint-staged": {
-    "{src,test}/**/*.ts": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "config": {
-    "commitizen": {
-      "path": "node_modules/cz-conventional-changelog"
-    }
-  },
-  "jest": {
-    "transform": {
-      ".(ts|tsx)": "ts-jest"
-    },
-    "testEnvironment": "node",
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
-    "moduleFileExtensions": [
-      "ts",
-      "tsx",
-      "js"
-    ],
-    "coveragePathIgnorePatterns": [
-      "/node_modules/",
-      "/test/"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 90,
-        "functions": 95,
-        "lines": 95,
-        "statements": 95
-      }
-    },
-    "collectCoverageFrom": [
-      "src/*.{js,ts}"
-    ]
-  },
-  "prettier": {
-    "semi": false,
-    "singleQuote": true
-  },
-  "commitlint": {
-    "extends": [
-      "@commitlint/config-conventional"
-    ]
+    "build": "tsc",
+    "test": "mocha -r ts-node/register src/**/*.spec.ts"
   },
   "devDependencies": {
```

Comment:
We might want to go through these and make sure that we don't have extra dependencies after removing browser support.